ci/build.yml

```diff
@@ -275,10 +275,31 @@ jobs:
           echo "Could not connect to Oracle, trying again..."
           sleep 1
         done
+
+
+        # Setting MAX_STRING_SIZE='EXTENDED' in order to enable specfying collation at column level.
+        # See 'Increasing the Maximum Size of VARCHAR2, NVARCHAR2, and RAW Columns in a PDB' at
+        # https://docs.oracle.com/en/database/oracle/oracle-database/19/refrn/MAX_STRING_SIZE.html#GUID-D424D23B-0933-425F-BC69-9C0E6724693C
+        #
+        cat <<EOT > set_max_string_size.txt
+         SHOW PARAMETER max_string_size;
+         SHUTDOWN IMMEDIATE;
+         ALTER PLUGGABLE DATABASE orclpdb1 OPEN UPGRADE;
+         ALTER SYSTEM SET max_string_size='EXTENDED';
+         @?/rdbms/admin/utl32k.sql;
+         @?/rdbms/admin/utlrp.sql;
+         SHUTDOWN IMMEDIATE;
+         STARTUP;
+         SHOW PARAMETER max_string_size;
+         ;
+        EOT
+        docker cp set_max_string_size.txt oracle:/home/oracle/set_max_string_size.txt
```

Comment:
You can pump the statements directly into sqlplus thus avoiding creation and copying of a file:
```
docker exec oracle bash -c 'sqlplus -L sys/'"$ORACLE_PWD"'@//localhost:'"$ORACLE_PORT"'/ORCLPDB1 as sysdba <<<"
         SHOW PARAMETER max_string_size;
         SHUTDOWN IMMEDIATE;
         ALTER PLUGGABLE DATABASE orclpdb1 OPEN UPGRADE;
         ALTER SYSTEM SET max_string_size='EXTENDED';
         @?/rdbms/admin/utl32k.sql;
         @?/rdbms/admin/utlrp.sql;
         SHUTDOWN IMMEDIATE;
         STARTUP;
         SHOW PARAMETER max_string_size;
         ;"'